### PR TITLE
[14.0][MIG] sale_contact_type

### PR DESCRIPTION
**Pull request by LudLaf**
Module migrated to v14

@pedrobaeza @Tardo please review.

@Tecnativa TT34016

### CODE REVIEW

File: sale_contact_type/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Sale Contact Type",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
```
Comment (pedrobaeza) [inline sale_contact_type/__manifest__.py:7]: This must be in the migration commit, not here.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Squash both existing migration commits (not the black one) into one

**Comment by LudLaf** [comment]
> Squash both existing migration commits (not the black one) into one

It's been sorted.

File: sale_contact_type/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply sale_contact_type/__manifest__.py:7]: @LudLaf

**Comment by LudLaf** [comment]
Changes made as requested @pedrobaeza

**Review by rousseldenis — APPROVED** [review]

File: sale_contact_type/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -1,2 +1 @@
-Adds a new type for contacts called 'Ordering contact'.
-Changes the customer domain in sale orders to show partners that are 'Ordering contact' or main partners (partners without parent) that haven't childs or without childs of type 'Ordering contact'.
+Adds a new contact's type which allows a company to mark (designate) at least one of its own contacts as the main 'ordering contact'. This action will help in finding out who to address when dealing with sales orders.
```
Comment (pedrobaeza) [inline sale_contact_type/readme/DESCRIPTION.rst]: This text is not complete. Second point is missing

File: sale_contact_type/readme/USAGE.rst [hunk 3]
```diff
@@ -1,2 +1 @@
-Create/modify a children contact (Contacts > Selected or Create one > Contacts & Addresses > Press on '+ Add') and set the type to 'Ordering contact'
-Create a Sale Order (Sales > Create) and select the contact created/modified in 'Customer' field.
+Create/modify the desired children's contact (Contacts > Select or Create > Contacts & Addresses > Press 'Add' button) and set type to 'Ordering contact'.
```
Comment (pedrobaeza) [inline sale_contact_type/readme/USAGE.rst]: Don't remove dashes, as they are steps to do, but better put `#. ` to indicate a numbered list

Comment (LudLaf) [reply sale_contact_type/readme/USAGE.rst]: Changes made.

**Comment by pedrobaeza** [comment]
There are 2 commits that don't belong here:

Selección_029

You may forget again to do `git fetch origin` before rebasing.

**Review by victoralmau — APPROVED** [review]
Code review ok

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**